cmd/svcat/output/table.go

```diff
@@ -37,26 +37,19 @@ const DefaultPageWidth = 80
 // value of our special column first, call SetColMinWidth and then add the
 // headers/rows.
 type ListTable struct {
-	table *tablewriter.Table
+	*tablewriter.Table
```

Comment:
Interesting, I didn't know you could ref it by "Table" later on like that.  This is why I still feel like such a newbie w.r.t. golang  - thanks!

Reply:
Using embedded fields to satisfy an interface is a really nice feature! 👍 

Sorry about pushing to your fork! Next time I send a gist or patch.

Reply:
No worries, just a "you dunce, just refer it to by `Table`" would have gotten the point across  ;-)